app/controllers/games_controller.rb

```diff
@@ -1,4 +1,34 @@
 class GamesController < ApplicationController
   # Add your GamesController code here
```

Comment:
To avoid repeating your code, you should use a before_action to set the game variable in both the show and update route. 

```ruby
class GamesController < ApplicationController
  before_action :set_game, only: [:show, :update]

  def show
    # show render
  end

  def update
    # update logic
  end

  private
    def set_game
      @game = Game.find(params[:id])
    end
end
```